2.0/0x300-Integrations/0x303-I3-Oracle.md

```diff
@@ -21,6 +21,13 @@ Category “I3” lists requirements related to the Oracle smart contract as one
 | **I3.4** | Verify that, when using Uniswap V3 TWAP as price oracle, liquidity is high enough and is distributed widely across most of the price range. |
 | **I3.5** | Verify that, the use a decentralized off-chain oracles unsusceptible to on-chain price manipulation attacks (e.g. Chainlink) is considered for low liquidity asset, ideally combining it with on-chain oracles to detect malicious values. |
 | **I3.6** | Verify that the value you are using has had enough time to be reported as invalid and has not been. |
+| **I3.7** | Verify that there is a check for stale prices when using Chainlink Oracle |
+| **I3.8** | Verify that there is a check for down l2 sequencer when using Chainlink Oracle  |
+| **I3.9** | Verify that NOT the same heartbeat is used for multiple price feeds when using Chainlink Oracle |
+| **I3.10** | Verify that the code deals with different price feeds having different decimal precision when using Chainlink Oracle  |
+| **I3.11** | Verify that the price feed address wherever it is located(hardcoded, deployment script) is pointing to the correct oracle price feed |
+| **I3.12** | Verify that the code handles calls to the oracle if they potentially revert when using Chainlink Oracle |
```

Comment:
Those need some rewording IMO because are not easily understandable, but that might be my English ;)
Will try to propose sth.

Reply:
Yes, It was hard for me to reframe them in a easily understandable way because they are not so common vulnerabilities and if you are a person who don't know them more in depth it will be hard to understand what are we saying with them. I believe the best thing will be that people read them, find out they don't know them good enough to understand and head to the article that I have listed about Chainlink Oracle Considerations where they can read further + take a look at some examples and be okay with the "Verify" list the other time.